source/whats-new.txt

```diff
@@ -32,4 +40,89 @@ What's New in 1.20
    Support for MongoDB Server v3.6 is removed in this release of the
    library.
 
-- Adds support for MongoDB Server v8.0.
\ No newline at end of file
+- Adds support for MongoDB Server v8.0.
+
+.. _php-lib-version-1.18:
+
+What's New in 1.18
+------------------
+
+- Adds a new GridFS API to make it more convenient to work with files using PHP's 
+  existing filesystem functions. The :ref:`MongoDB\GridFS\Bucket::registerGlobalStreamWrapperAlias() <api-gridfs-bucket-register-global>` 
+  method may be used to register a global alias for a GridFS bucket. After 
+  doing so, files within that bucket can be accessed using only a file URI 
+  (e.g. "gridfs://mybucket/hello.txt"). A demonstration of this API can be found
+  in the `gridfs_stream_wrapper.php <https://github.com/mongodb/mongo-php-library/blob/1.18.0/examples/gridfs_stream_wrapper.php>`__ 
+  example script.
+
+- Adds :ref:`addSubscriber() <api-client-add-subscriber>` and 
```

Comment:
S: there's a :phpmethod: directive that you can use to link!
```suggestion
- Adds :phpmethod:`addSubscriber()` and 
```

Reply:
Applies to all - you also don't need to add the ref anchors to the API pages (add_logger.txt, remove_logger.txt, etc)